brambling/templates/brambling/dashboard.html

```diff
@@ -4,43 +4,68 @@
 
 {% block title %}Your events – {{ block.super }}{% endblock %}
 
+{% block content %}
+	{% if not request.user.is_authenticated or not organizations and not admin_events %}
+		{% if not request.COOKIES.hideCall %}
+			<div class="masthead clearfix masthead-brand orange-barred">
+				<div class="row">
+					<div class="col-sm-6">
+						<p class="text-jumbo"><strong class="text-accent">Dancerfly</strong> is an easy-to-use dance event registration and ticketing system.</p>
+					</div>
+					<div class="col-sm-6">
+						<p class="text-jumbo">Start using Dancerfly for your <strong class='text-accent'>organization</strong> or <strong class='text-accent'>event</strong> today!</p>
+						<p>
+							<a class='btn btn-lg btn-primary-light'>Learn more <i class='fa fa-chevron-right'></i></a>
+							<a id='hideCall' class='btn btn-lg btn-link masthead-link'>Hide</a>
+						</p>
+					</div>
+				</div>{# /.row #}
+			</div>
+		{% endif %}
```

Comment:
Oh, hm. contrib hints requires a user. It should probably also accept a session.